### ISSUE TITLE
Virtuality options drop-down for MariaDB 

### ISSUE THREAD

**Issue by OlafvdSpek** [phpmyadmin/phpmyadmin#15729]
```
There are two types of generated columns:

PERSISTENT (a.k.a. STORED): This type's value is actually stored in the table.
VIRTUAL: This type's value is not stored at all. Instead, the value is generated dynamically when the table is queried. This type is the default.
```

So it doesn't make sense to have three options..

https://mariadb.com/kb/en/generated-columns/

![pma](https://user-images.githubusercontent.com/2218936/71640520-e7d82580-2c8b-11ea-8c80-ef88fee63f73.PNG)

**Comment by victorphoenix3** [comment]
@williamdes what needs to be done here?

**Comment by williamdes** [comment]
@victorphoenix3 I would agree to remove an option and maybe reword persistent option

**Comment by victorphoenix3** [comment]
@williamdes The PR #14966 added the 'stored' option for mariadb .Should I revert this change?

**Comment by OlafvdSpek** [comment]
No, https://github.com/phpmyadmin/phpmyadmin/pull/14966#issuecomment-465575141

**Comment by williamdes** [comment]
@OlafvdSpek what needs to be done ?

**Comment by victorphoenix3** [comment]
@OlafvdSpek  Currently we show only 'stored' for mysql and both 'persistent' and 'stored' for mariadb.

**Comment by OlafvdSpek** [comment]
Yes, it seems `stored` is required to fix https://github.com/phpmyadmin/phpmyadmin/issues/14957 though I don't understand why. Is MariaDB returning `stored` rather then `persistent`?

**Comment by victorphoenix3** [comment]
@williamdes  we are returning `stored` even for columns generated as `persistent`.
So, maybe we should remove the `persistent` option and just keep `stored` and `virtual`.
I will check the codebase again to make sure nothing breaks.

**Comment by OlafvdSpek** [comment]
Does MariaDB 10.1 support `stored`?

**Comment by victorphoenix3** [comment]
stored is only supported by 10.2.1 and later.
I will apply a check on the version.
